Services/CO.CDP.OrganisationInformation.Persistence.Tests/DatabaseAuthenticationKeyRepositoryTest.cs

```diff
@@ -62,10 +62,10 @@ public async Task ItFindsSavedAuthenticationKeyByNameKeyAndOrganisationId()
         using var repository = AuthenticationKeyRepository();
         var organisation = GivenOrganisation();
         var key = Guid.NewGuid().ToString();
-        var authenticationKey = GivenAuthenticationKey(key: key, organisation: organisation);
+        var authenticationKey = GivenAuthenticationKey(name: "fts-key", key: key, organisation: organisation);
```

Comment:
Details that are relevant to the test case should be visible in the test and not hidden. This name is used to find the record later, so I made it visible so it's clear where the name comes from.